### PR TITLE
Updated translation

### PR DESCRIPTION
**Pull request by dinolaz**

### CODE REVIEW

File: src/musikcube/data/locales/uk_UA.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "browse_title_directory_tracks": "пісні в %s",
     "browse_title_genres": "жанри",
     "browse_title_playlists": "грайлисти",
-    "browse_title_tracks": "пісні (сортувати за %s)",
+    "browse_title_tracks": "пісні (сортувати за альбомом)",
```
Comment (clangen) [inline src/musikcube/data/locales/uk_UA.json]: This line looks a little suspicious -- I think there's supposed to be a `%s` here. Is there a reason this change was made?

**Review by clangen — COMMENTED** [review]
Looks good, but I have a question :)

File: src/musikcube/data/locales/uk_UA.json (hunk 1 above)
Comment (dinolaz) [reply src/musikcube/data/locales/uk_UA.json]: Oh sorry, i fixed it in new commits

**Comment by clangen** [comment]
Sorry for the delay here -- real life has taken up a lot of time lately. Thank you for the contribution! :)

**Pull request merged**